lib/timezone/timezone.ex

```diff
@@ -77,6 +77,19 @@ defmodule Timex.Timezone do
   def name_of("Y"),    do: name_of(-12)
   def name_of("Z"),    do: "UTC"
   def name_of("UT"),   do: "UTC"
+  def name_of("+0330"), do: "Asia/Tehran"
```

Comment:
Cool. I have zero urgency around this. I knew the odd offsets existed and that's it. 

I'll try to update in the next couple evenings or so.